pkg/ctl/create/utils.go

```diff
@@ -53,26 +53,17 @@ func checkVersion(cmd *cmdutils.Cmd, ctl *eks.ClusterProvider, meta *api.Cluster
 	return nil
 }
 
-func hasInstanceType(nodeGroup *api.NodeGroup, hasType func(string) bool) bool {
-	if hasType(nodeGroup.InstanceType) {
-		return true
-	}
-	if nodeGroup.InstancesDistribution != nil {
-		for _, instanceType := range nodeGroup.InstancesDistribution.InstanceTypes {
-			if hasType(instanceType) {
-				return true
-			}
+func showDevicePluginMessageForNodeGroup(nodeGroup *api.NodeGroup, installNeuronPlugin bool) {
+	if eks.HasInstanceType(nodeGroup, utils.IsNeuronInstanceType) {
```

Comment:
nit: the instance type is inferentia so I would rename `IsNeuronInstanceType` to `IsInfInstanceType`,`IsInferentiaType`or similar